### DISCUSSION TITLE
How to Disable a single default Alt-Enter key binding?

### DISCUSSION THREAD

**Discussion by kulak** [wezterm/wezterm#2891] [Q&A] [answered]
Alt-Enter is bound to maximize the screen, but it hides Alt-Enter of `broot` program and under Gnome we have Alt-F10 for all applications.

So, I'd like to disable just that key binding, but documentation has instructions only for all or nothing.

**Comment by yutkat** [comment, marked as answer]
If you want to disable the default Alt+Enter, you can set the following.

```lua
local wezterm = require 'wezterm'

return {
  keys = {
    {
      key = 'Enter',
      mods = 'ALT',
      action = wezterm.action.DisableDefaultAssignment,
    },
  },
}
```
https://wezfurlong.org/wezterm/config/keys.html#configuring-key-assignments
https://wezfurlong.org/wezterm/config/lua/keyassignment/DisableDefaultAssignment.html

The key bindings that are set by default can be found in this document.
https://wezfurlong.org/wezterm/config/default-keys.html

If it's too annoying to disable, you can disable all keybindings with this setting.
```lua
return {
  disable_default_key_bindings = true,
}
```

**Comment by LorenzoBettini** [comment]
This works as expected, however, the Command Palette does not get udpated and still shows Alt+ENTER.
Is that intentional?
